### PR TITLE
AnalyzerCommand: Fix calculating the total project count

### PR DESCRIPTION
**Pull request by sschuberth**
Signed-off-by: Sebastian Schuberth <sebastian.schuberth@bosch.io>

### CODE REVIEW

File: cli/src/main/kotlin/commands/AnalyzerCommand.kt [hunk 1]
```diff
@@ -228,16 +228,19 @@ class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine depende
...
                 println("Found ${files.size} $manager project(s) at:")
+
                 files.forEach { file ->
                     val relativePath = file.toRelativeString(inputDir).takeIf { it.isNotEmpty() } ?: "."
                     println("\t$relativePath")
                 }
             }
 
-            println("Found ${filesPerManager.size} project(s) in total.")
+            println("Found $count project(s) from ${filesPerManager.size} package manager(s) in total.")
```
Comment (mnonnenmacher) [inline cli/src/main/kotlin/commands/AnalyzerCommand.kt:243]: You could use `filesPerManager.values.sumOf { it.size }` here to get rid of the `count` variable.

Comment (sschuberth) [reply cli/src/main/kotlin/commands/AnalyzerCommand.kt:243]: Correct, but I didn't like so much to iterate over `filesPerManager` twice.

Comment (mnonnenmacher) [reply cli/src/main/kotlin/commands/AnalyzerCommand.kt:243]: I think performance wise this is completely irrelevant, and I don't like the `count` variable :) But I'll leave it up to you.

**Review by mnonnenmacher — APPROVED** [review]

File: cli/src/main/kotlin/commands/AnalyzerCommand.kt (hunk 1 above)
Comment (mnonnenmacher) [reply cli/src/main/kotlin/commands/AnalyzerCommand.kt:243]: Too late now anyway, since I missed that auto-merge was on.

Comment (sschuberth) [reply cli/src/main/kotlin/commands/AnalyzerCommand.kt:243]: It's never too late for a follow-up 😉 I agree performance-wise it does not matter, but it just does not seem right to me to recalculate something that is already partly know.

**Pull request merged**